src/videouplinkbandwidthpolicy/DefaultSimulcastUplinkPolicy.ts

```diff
@@ -0,0 +1,379 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import Logger from '../logger/Logger';
+import Maybe from '../maybe/Maybe';
+import AsyncScheduler from '../scheduler/AsyncScheduler';
+import SimulcastLayers from '../simulcastlayers/SimulcastLayers';
+import SimulcastTransceiverController from '../transceivercontroller/SimulcastTransceiverController';
+import DefaultVideoAndEncodeParameter from '../videocaptureandencodeparameter/DefaultVideoCaptureAndEncodeParameter';
+import VideoStreamDescription from '../videostreamindex/VideoStreamDescription';
+import VideoStreamIndex from '../videostreamindex/VideoStreamIndex';
+import BitrateParameters from './BitrateParameters';
+import ConnectionMetrics from './ConnectionMetrics';
+import SimulcastUplinkObserver from './SimulcastUplinkObserver';
+import SimulcastUplinkPolicy from './SimulcastUplinkPolicy';
+
+const enum ActiveStreams {
```

Comment:
Yes, it was already part of the existing `SimulcastUplinkPolicy`. As stated in description, `SimulcastUplinkPolicy` is renamed to `DefaultSimulcastUplinkPolicy` and `SimulcastUplinkPolicy` is converted into an interface. Further I think `ActiveStreams` is only used internally only in `DefaultSimulcastUplinkPolicy` class; thats why it might have not been exported out in a separate file. I have separated other enum for e.g. `SimulcastLayers` into a separate file as you said.